tests/Kernel/TenderRenderTest.php

```diff
@@ -0,0 +1,254 @@
+    $this->assertCount(1, $actual);
+
+    // Check Department/s label for multiple department values.
+    $node->set('oe_departments', 'http://publications.europa.eu/resource/authority/corporate-body/ABEC')->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $expected_values['description'] = new PatternAssertState(new FieldListAssert(), [
+      'items' => [
+        [
+          'label' => 'Reference',
+          'body' => 'Call for tenders reference',
+        ], [
+          'label' => 'Opening date',
+          'body' => $opening_date->format('d F Y'),
+        ], [
+          'label' => 'Deadline date',
+          'body' => $deadline_date->format('d F Y, H:i (T)'),
+        ], [
+          'label' => 'Department',
+          'body' => 'Audit Board of the European Communities',
+        ],
+      ],
+    ]);
+    $assert->assertPattern($expected_values, $html);
+
+    // Check status Closed label and background.
+    $deadline_date = (clone $static_time)->modify('- 2 days');
+    $node->set('oe_tender_deadline', $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT))->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $expected_values['meta'] = 'Call status: Closed';
+    $deadline_date->setTimeZone(new \DateTimeZone('Australia/Sydney'));
+    $expected_values['description'] = new PatternAssertState(new FieldListAssert(), [
+      'items' => [
+        [
+          'label' => 'Reference',
+          'body' => 'Call for tenders reference',
+        ], [
+          'label' => 'Opening date',
+          'body' => $opening_date->format('d F Y'),
+        ], [
+          'label' => 'Deadline date',
+          'body' => $deadline_date->format('d F Y, H:i (T)'),
+        ], [
+          'label' => 'Department',
+          'body' => 'Audit Board of the European Communities',
+        ],
+      ],
+    ]);
+    $assert->assertPattern($expected_values, $html);
+
+    $crawler = new Crawler($html);
+    $actual = $crawler->filter('span.tender-status.ecl-label.ecl-u-text-uppercase.ecl-label--highlight');
+    $this->assertCount(1, $actual);
+
+    // Check Deadline date is striked when tender is close.
+    $actual = $crawler->filter('dd.ecl-description-list__definition > .ecl-u-type-strike');
+    $this->assertCount(1, $actual);
+
+    // Check status Upcoming label and background.
+    $opening_date = (clone $static_time)->modify('+ 10 days');
+    $deadline_date = (clone $static_time)->modify('+ 5 days');
+    $node->set('oe_tender_opening_date', $opening_date->format('Y-m-d'))->save();
+    $node->set('oe_tender_deadline', $deadline_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT))->save();
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $deadline_date->setTimeZone(new \DateTimeZone('Australia/Sydney'));
```

Comment:
Why do we need to set the timezone every time?

Reply:
It is because we are creating a new $deadline_date object before, for example: https://github.com/openeuropa/oe_theme/blob/3e17aef4bd08e07947cc78fb57e8f7c55edf0e3e/tests/Kernel/TenderRenderTest.php#L192